packages/core/src/configuration.ts

```diff
@@ -139,7 +139,8 @@ const chart: BaseChartOptions = {
 		selectedGroups: []
 	},
 	color: {
-		scale: null
+		scale: null,
+		gradientEnabled: false
```

Comment:
again, we need this to be:
```js
{
  color: {
    gradient: {
       enabled: false
    }
  }
}
```

since we possibly will have to add more flags/customizables in the future for gradients.